spec/git-repository-spec.js

```diff
@@ -404,7 +404,7 @@ describe('GitRepository', () => {
     it('subscribes to all the serialized buffers in the project', async () => {
       atom.project.setPaths([copyRepository()]);
 
-      await atom.workspace.open('file.txt');
+      await atom.workspace.open('a.txt');
```

Comment:
Is this change necessary?

Reply:
Yes, without this a spec fails because someone renamed a file without editing this.

Reply:
That doesn't sound right. Surely the tests would be consistently failing for this reason on master if that were the case. `atom.workspace.open` should work even on files that don't exist, and there are several other references to `file.txt` anyway.

Reply:
I need to take a closer look. It is in `working-dir` correct?

Reply:
a.txt seems to have no content...

Reply:
Maybe this test failed just due to flakiness? Other branches and PRs do pass with this saying `file.txt`.

Reply:
Ah. Will undo this.

Reply:
That sounds most likely. Is there a link to the test results that did fail with it?

Reply:
No. I'm running a clone of the pipeline on my version right now though.

Reply:
Resolving because this is outdated...